### PR TITLE
Fix delete button

### LINKED ISSUE #489: Delete layout error <danger-button>
Since 4.28.x of Laravel Nova, in the confirm panel to delete the layout, it doesn't work.

2023-11-20 at 6  58 44@2x

It looks like <danger-button> component is not rendering.

### PR DESCRIPTION
**Pull request by nbolender**
Looks like Nova 4.28 made some changes that broke the delete button. This fixes #489 

Let me know if I need to change the way I made this PR. It works, but not sure if I followed the right procedure for dependencies and compiling.

### CODE REVIEW

**Comment by jsmit99** [comment]
Why won't these fixes be merged? CC @toonvandenbos

File: package.json
Comment (wize-wiz) [inline package.json:1]: Where does [laravel-nova-ui](https://socket.dev/npm/package/laravel-nova-ui/overview/0.4.12) come from and do you perhaps know why this wasn't a problem before?

**Comment by wize-wiz** [comment]
@voidgraphics This can be merged, I've tested it and compiles without problems. laravel-nova-ui seems to be a bundle of various elements unknown to anyone, but anyhow ;)

**Comment by jsmit99** [comment]
Any update on this? @voidgraphics 

As it's still broken without this patch.

**Comment by voidgraphics** [comment]
Thank you, released in `v1.0.14`.

**Pull request merged**